vendor/bat-native-ads/src/bat/ads/internal/eligible_ads/sample_ads_unittest.cc

```diff
@@ -0,0 +1,153 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "bat/ads/internal/eligible_ads/sample_ads.h"
+
+#include "base/guid.h"
+#include "bat/ads/internal/bundle/creative_ad_notification_info.h"
+#include "bat/ads/internal/bundle/creative_ad_notification_unittest_util.h"
+#include "bat/ads/internal/eligible_ads/eligible_ads_util.h"
+#include "bat/ads/internal/unittest_util.h"
+
+// npm run test -- brave_unit_tests --filter=BatAds*
+
+namespace ads {
+
+TEST(BatAdsSampleAdsTest, CalculateNormalisingConstantWithEmptyAds) {
+  // Arrange
+  CreativeAdNotificationPredictorMap ads;
```

Comment:
nit: predictors instead of ads, same for templated functions

Reply:
fixed